packages/flutter_test/lib/src/finders.dart

```diff
@@ -548,26 +572,66 @@ abstract class MatchFinder extends Finder {
 }
 
 class _TextFinder extends MatchFinder {
-  _TextFinder(this.text, { bool skipOffstage = true }) : super(skipOffstage: skipOffstage);
+  _TextFinder(
+    this.text, {
+    this.skipRichText = true,
+    bool skipOffstage = true,
+  }) : super(skipOffstage: skipOffstage);
 
   final String text;
 
+  /// Whether standalone [RichText] widgets should be skipped or not.
+  ///
+  /// Defaults to `true`.
+  ///
+  /// If enabled, only [Text] widgets will be considered. These of course also
+  /// insert a [RichText] widget, however, that is ignored and so are all
+  /// standalone [RichText] widgets.
+  /// If disabled, only [RichText] widgets will be considered. This *implicitly*
+  /// considers [Text] widgets as well since they always insert a [RichText]
+  /// child.
+  ///
+  /// In either case, [EditableText] widgets will also be considered.
+  final bool skipRichText;
```

Comment:
maybe we should name this positively: findRichText or something like that.

Reply:
@goderbauer I thought about this as well, but I wanted to match `skipOffstage`. What do you think?

Reply:
I think skipOffstage is also poorly named. Since we are adding something new, let's give it a better name.